.dockerignore

```diff
@@ -0,0 +1,10 @@
+src/__pycache__
+src/*.pyc
+src/*.pyo
+src/*.pyd
+src/.Python
+src/venv
+src/pip-log.txt
+src/pip-delete-this-directory.txt
+src/.cache
+src/.git
```

Comment:
.git, venv and pip-* are not in src.

I'm not even sure what the 2 pip files are, I've never seen them before